Makefile

```diff
@@ -2,7 +2,7 @@ DEBUG=0
 DEBUGGER=0
 SPLIT_UP_LINK=0
 CORE_DIR := src
-TARGET_NAME := mame2003_plus
+TARGET_NAME := mame2003-plus
```

Comment:
renamed again?

Reply:
nope was done be @dankcushions  so he could build this wont be changing